go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/google/uuid v1.2.0
 	github.com/gorilla/mux v1.7.4
 	github.com/spf13/cobra v1.0.0
-	github.com/tektoncd/pipeline v0.20.0
+	github.com/tektoncd/pipeline v0.20.1-0.20210128163741-1eca890e74c9
```

Comment:
Why is this an untagged version of pipeline?

Reply:
I bumped it to the latest so we'd get the proper k8s deps in the `require` block

https://github.com/tektoncd/pipeline/pull/3724